test/e2e/tests/tokens/increase-token-allowance.spec.js

```diff
@@ -272,10 +272,14 @@ describe('Increase Token Allowance', function () {
       tag: 'button',
       text: 'Next',
     });
-    driver.waitForSelector({
+    await driver.waitForSelector({
       css: '.box--display-flex > h6',
       text: `10 TST`,
     });
+    await driver.waitForSelector({
+      tag: 'h6',
+      text: '0.000062 ETH',
+    });
```

Comment:
here we await for the correct gas estimate. This way we ensure that the increaseAllowance tx is successful and we can proceed with the transferFrom, from account 2

Reply:
thank you for the review @legobeat :bowing_woman:  I think in this case is safe to rely on the default timeout, since I've observed that the gas update is really a matter of ms, so the default timeout seems more than enough.
But let me know if you think otherwise and we could add it :pray: 

```
async waitForSelector(
    rawLocator,
    { timeout = this.timeout, state = 'visible' } = {}
```

![Screenshot from 2024-05-23 10-11-42](https://github.com/MetaMask/metamask-extension/assets/54408225/59a9b52b-218c-4bf2-a7cb-855cdbb6ab08)

Reply:
:information_source: after talking with Lego, we've agreed to merge this PR but we could have another task to look into explicit timeouts more generally